packages/web/src/components/add-funds-modal/AddFundsModal.tsx

```diff
@@ -81,7 +82,7 @@ export const AddFundsModal = () => {
       onClosed={handleClosed}
       bodyClassName={styles.modal}
       useGradientTitle={false}
-      dismissOnClickOutside
+      dismissOnClickOutside={!isTransferring}
```

Comment:
Do we need to disable this while the Stripe modal is open as well? Or does their backdrop intercept all the clicks?